packages/analytics/README.md

```diff
@@ -1,5 +1,39 @@
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
+
 # cloud-sdk-analytics
 
 This package contains the code we use for collecting usage data.
 Please note that usage analytics is disabled by default, i.e. you need to explicitly opt-in for data to be collected.
-For more information on how to opt-in and which data is collected, please refer to [this document](https://github.com/SAP/cloud-sdk-cli/blob/main/usage-analytics.md).
+For more information on how to opt-in and which data is collected, please refer to [this document](https://github.com/SAP/cloud-sdk-js/blob/main/packages/analytics/usage-analytics.md).
+
+
+<!-- This block is inserted by scripts/copy-generic-readme.ts and not oclif like the commands block --> 
+<!--genericPart-->
```

Comment:
Maybe instead of an additional comment give it a more speaking name? E. g. `<!-- sap-cloud-sdk-common-readme -->`. In addition you could call the script `replace-common-readme.ts` to correspond to this.

Reply:
BTW, I think an additional comment can make sense to warn us not to change the generated part. In that case, it should be part of what is generated. But I think this might be overkill.

Reply:
I added a `Do not adjust it manually part`